bsip-0086.md

```diff
@@ -0,0 +1,119 @@
+    BSIP: 0086
+    Title: Share market fees to the network
+    Author: Abit More <https://github.com/abitmore>
+    Status: Draft
+    Type: Protocol
+    Created: 2019-11-29
+    Discussion: https://github.com/bitshares/bsips/issues/194
+    Worker: TBD
+
+# Abstract
+
+This BSIP proposes a protocol change: when every trade happens, if there are
+market fees generated, a portion (E.G. 5% or 10%) of the market fees goes to
+the network.
+
+# Motivation
+
+The network needs more tools to generate income to support its development.
+
+# Rationale
+
+A major activity in the network is trading assets. The asset owners make
+profits in the form of market trading fees. Currently, the main tools for the
+network to generate income from asset trading activities are order creation
+fee and order cancellation fee. It seems reasonable that the asset owners
+share some profits (a part of market fees) to the network to support its
+development.
+
+The cut of market fees can go to committee-account's vesting balances.
+As a supporting measure, committee-account should be exempted from
+white-listing restrictions, so that it is able to claim the vesting balances
+and sell them for core token or other tokens to pay for development later.
+
+# Specifications
+
+## Protocol upgrade
+
+A time will need to be scheduled for applying the change. In this document,
+terms "before the protocol upgrade", "at the protocol upgrade" and "after
+the protocol upgrade" may or may not be used to indicate things happen before
+the scheduled time, at the scheduled time and after the scheduled time.
+
+## New global parameter
+
+Add a new global parameter `market_fee_network_percent` which can be updated
+by the committee only after the protocol upgrade.
+Initial value of that parameter is `0`.
+Valid range of that parameter is `[0, 100%]`.
```

Comment:
Parameter should be `[0, 25%]`

a) we cant force against private business or even be able to take their entire income.
b) existing 3rd parties and new on-boarding businesses/asset owners will feel more safe knowing that maximum cut they can pay is 25% from their market fee.

Reply:
I'd like to hear others' opinions. @sschiessl-bcp @MichelSantos @bitcrab please?

Reply:
Well since you asked I added [my opinion](https://github.com/bitshares/bsips/issues/194#issuecomment-568225147) into the issue to avoid its getting lost

Reply:
I agree to limit the parameter to ` [0, 25%]`, 25% is high enough.

Reply:
Does the network percentage of transaction fees have any limit?

The one thing that came to mind is that a fixed upper limit might gateways to do worst case planning, instead of opening communication with committee to agree on feasible values.

Reply:
@sschiessl-bcp what kind of worst case planning? upper is just to tell public that system at most can charge so much on market fees.

As GDEX founder to me a 5%-10% market fee sharing to system is totally acceptable, even 25% is discussible，not aware of other gateways' opinions.

Reply:
With worst case planning I mean: If the fee sharing limit is set to 20%, then gateways could simply adjust the fee such that the remaining 80% reflect the actually intended fee, and if the actual fee sharing % is set lower than the limit, gateways are simply making a bonus. In other words: The limit would be to the downside of the user.

This is certainly a thoughtplay, but our gateways have shown worst-case behavior lately.

Reply:
this is just as the tax logic, the tax paid by service provider to goverment always come from the service buyer. no problem.